cmd/eksctl/main.go

```diff
@@ -18,67 +18,67 @@ import (
 	"github.com/weaveworks/eksctl/pkg/ctl/utils"
 )
 
-var rootCmd = &cobra.Command{
-	Use:   "eksctl [command]",
-	Short: "a CLI for Amazon EKS",
-	Run: func(c *cobra.Command, _ []string) {
-		if err := c.Help(); err != nil {
-			logger.Debug("ignoring error %q", err.Error())
-		}
-	},
+func addCommands(rootCmd *cobra.Command, flagGrouping *cmdutils.FlagGrouping) {
+	rootCmd.AddCommand(versionCmd(flagGrouping))
+	rootCmd.AddCommand(create.Command(flagGrouping))
+	rootCmd.AddCommand(delete.Command(flagGrouping))
+	rootCmd.AddCommand(get.Command(flagGrouping))
+	rootCmd.AddCommand(update.Command(flagGrouping))
+	rootCmd.AddCommand(scale.Command(flagGrouping))
+	rootCmd.AddCommand(drain.Command(flagGrouping))
+	rootCmd.AddCommand(utils.Command(flagGrouping))
+	rootCmd.AddCommand(completion.Command(rootCmd))
 }
 
-func init() {
+func main() {
+	cobra.EnableCommandSorting = false
 
-	var colorValue string
+	rootCmd := &cobra.Command{
+		Use:   "eksctl [command]",
+		Short: "a CLI for Amazon EKS",
```

Comment:
Now that we've changed the repo's description to "The official CLI for AWS EKS", what do you think about mentioning "official" here also?

Reply:
Yes, I'm all for consistency :)